Godeps/Godeps.json

```diff
@@ -3929,11 +3929,6 @@
 			"Rev": "225eecce40065518f2eabc3b52020365db012384"
 		},
 		{
-			"ImportPath": "k8s.io/kubernetes/cmd/libs/go2idl/args",
```

Comment:
I don't think so. When you're doing a rebase, if there's a .go file in origin that references a package that no longer exists, you should be able to correct the .go file and then update Godeps.json in the same commit to remove the references to the package that's no longer used.